### PR TITLE
Create appscale-datastore package

### PR DESCRIPTION
**Pull request by cdonati**

### CODE REVIEW

**Comment by cdonati** [comment]
https://ocd.appscale.com:8080/job/Daily%20Build/2191/
With updated migration job: https://ocd.appscale.com:8080/job/Migration/658/

File: AppDB/delete_all_records.py [hunk 1]
```diff
@@ -20,93 +16,18 @@
...
-
-      first_key = entities[-1].keys()[0]
-      start_inclusive = False
-    except AppScaleDBConnectionError, connection_error:
-      logging.error("ERROR while deleting data from {0}.".format(connection_error))
-      logging.error(connection_error.message)
-      time.sleep(_BACKOFF_TIMEOUT)
 
 if __name__ == "__main__":
+  logging.basicConfig(format=LOG_FORMAT, level=logging.INFO)
```
Comment (menivaitsi) [inline AppDB/delete_all_records.py]: That is reasonable.

**Comment by menivaitsi** [comment]
Is this branch blocking something else?
It has conflicts that need to be resolved.

**Comment by cdonati** [comment]
It's not urgent. I'm going to wait for https://github.com/AppScale/appscale/pull/2222 before pushing a merge commit since it was brought up during the demo.

**Comment by cdonati** [comment]
https://ocd.appscale.com:8080/job/Daily%20Build/2284/
https://ocd.appscale.com:8080/job/Migration/727/

**Comment by menivaitsi** [comment]
lgtm

**Pull request merged**